### PR TITLE
Create v2ui (part 1)

### PR DESCRIPTION
**Pull request by taoalpha**

### CODE REVIEW

**Comment by polymorpher** [comment]
Let's adjust after https://github.com/polymorpher/one-wallet/pull/306 is merged

File: code/client/src/pages/Create.jsx [hunk 1]
```diff
@@ -162,14 +165,14 @@ const Create = ({ expertMode, showRecovery }) => {
   }, [])
 
   return (
-    <>
+    <div className={v2ui ? 'wallet-creation-flow' : ''}>
```
Comment (polymorpher) [inline code/client/src/pages/Create.jsx:168]: possible to avoid custom class?

File: code/client/src/components/OtpSetup.jsx [hunk 2]
```diff
@@ -27,6 +28,27 @@ export const OtpSetup = ({ isMobile, otpRef, otpValue, setOtpValue, name }) => {
   )
 }
 
+export const OtpSetup2 = ({ isMobile, otpRef, otpValue, setOtpValue, name }) => {
```
Comment (polymorpher) [inline code/client/src/components/OtpSetup.jsx:31]: need to add the autocomplete stuff from master later in `newui`

**Review by polymorpher — APPROVED** [review]

File: code/client/src/pages/Create.jsx (hunk 1 above)
Comment (taoalpha) [reply code/client/src/pages/Create.jsx:168]: definitely, will address that in the newui pr

**Pull request merged**